### ISSUE TITLE
0 peers 0 percent downloading

### ISSUE THREAD

**Issue by EaterComputer** [haugene/docker-transmission-openvpn#1345]
I'm new to docker. I'm running this on docker windows subsystem for linux using kitematic. From what I can tell in the container logs, it is accessing my vpn (surfshark) and I can access the webui. What could be going wrong?

**Comment by haugene** [comment]
Sorry about the late reply here. Did you get anywhere on this? 

Dns is a well known culprit. Can you share your docker run command and logs?

**Comment by randomodbuild** [comment]
Having this issue as well. No blocklist in place, and I am using NordVPN, even on the P2P category (I have also tried the normal setting). Very strange because the setup was working just fine a few weeks ago. But same issue, shows 0 percent and downloading from 0 of 0 peers. Even using a checkmyip torrent does not show up. Not sure if this matters but under the Network tab, under the listening port, it says closed.

**Comment by haugene** [comment]
@randomodbuild Try a clean mount point. Any difference? Just point the volume mount for /data for another temporary folder. Or remove it so you don't mount it. This should run with Transmission in a "fresh" state. If it works it could be something in your settings.

Also. Post your docker run command / docker-compose file plus your logs. Very hard to debug without it.

**Comment by randomodbuild** [comment]
Hey, sorry about this, I fixed my problem. I just had to change the formatting of my labels in the docker compose. They used to have a hyphen in front of each label and an equal sign before each variable (with no space in between), and I had to change it to remove the hyphen and have a semi-colon and space instead of the equal sign. I'm sure there is a better way of saying this. Basically, I used Smart Home Beginner's github transmission-vpn snippet found here: https://github.com/htpcBeginner/docker-traefik/blob/master/docker-compose-t2.yml
